### DISCUSSION TITLE
Switch 组件

### DISCUSSION THREAD

**Discussion by yangjinjun3** [jdf2e/nutui-react#16] [General]
功能、基本交互测试
...
文档完整性验证
...
...
props 所有属性测试
1、无法自定义class和style
2、change事件不起作用
...
props 异步修改组件是否动态更新
...
...
emits 所有事件测试
...
...
npm run build 子组件是否被构建
1、没有在 nutui.react.ts 中导出
...
其它

**Comment by irisSong** [comment]
1、第三个demo，change时间没有触发
